crates/xilem_masonry/src/view/flex.rs

```diff
@@ -121,7 +130,6 @@ impl ElementSplice for FlexSplice<'_> {
     }
 
     fn len(&self) -> usize {
-        // This is not correct because of the spacer items. Is `len` actually needed?
-        self.element.len() - self.ix
+        self.ix
```

Comment:
Sure! I suspect that we'll probably just remove `len`, anyway; but this is more correct

Reply:
Yeah probably makes sense, doesn't seem to be needed anymore (I'll probably look into this more deeply, whether that is indeed the case).

Reply:
It's used in old `xilem::view::List`, so I *guess* we still need this, in case we don't exactly know the size of a `ViewSequence` in the view logic (e.g. lazy loading of items in a list, not sure currently whether that should happen in the xilem reactive layer?)

Reply:
I don't think its use in `List` is necessary, but I haven't remade the `List` ViewSequence yet, so I'm not certain